pkg/kubelet/kuberuntime/labels.go

```diff
@@ -0,0 +1,276 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kuberuntime
+
+import (
+	"encoding/json"
+	"strconv"
+
+	"github.com/golang/glog"
+	"k8s.io/kubernetes/pkg/api"
+	kubecontainer "k8s.io/kubernetes/pkg/kubelet/container"
+	"k8s.io/kubernetes/pkg/kubelet/types"
+	"k8s.io/kubernetes/pkg/kubelet/util/format"
+	kubetypes "k8s.io/kubernetes/pkg/types"
+)
+
+const (
+	// TODO: move those label definitions to kubelet/types/labels.go
```

Comment:
Discussed with @yujuhong offline on this very topic. My stand (I could be very wrong) on this is that 

* We should change those labels to follow Kubernetes's format in general.
* This kind of labels should be the implementation detail to Kubelet, not the API defining between Kubelet and CRIs. In today's docker / rkt support, we are using some of those labels in our system, but the logic is all in kubelet side. 
* If above statement is true and the detail of label format is encapsulated at Kubelet side, we can change them later.
* Let's mark this as TODO, and reiterate this before we switch Kubelet to use the new CRI.

WDYT?